docs/Getting-Started-Guides/On-Premises-FPGA-Getting-Started/Initial-Setup/Xilinx-XDMA-Template.rst

```diff
@@ -198,11 +198,13 @@ Now, let's attach your |fpga_name|_ FPGA(s) to your Run Farm Machines:
 
 11. Uncheck "Verify" and click OK.
 
-12. When programming the configuration memory device is completed, power off your machine fully (i.e., the FPGA should completely lose |dip_switch_extra|
+12. Right-click on your FPGA and clock "Boot from Configuration Memory Device".
```

Comment:
@sagark Was this ever needed?

Reply:
What does this button do? My impression is that you can just program the MCS and that's it (it should always boot from the MCS)

Reply:
```suggestion
12. Right-click on your FPGA and click "Boot from Configuration Memory Device".
```

Reply:
When I didn't clock this button & did a cold-reboot, the `lspci` didn't recognize the FPGAs.